.mergify.yml

```diff
@@ -0,0 +1,15 @@
+pull_request_rules:
+  - name: Automatic merge on CI success and review
+    conditions:
+      - "status-success=ci/circleci: build"
```

Comment:
I don't see circle even running here so I think this will block everything. May need to turn circle on for the repo or something

Reply:
Circle is running, I set it up earlier. It hasn't reported yet because or ordering. Once this goes in, the next PR will be fine.